functional-test/src/test/groovy/org/rundeck/util/gui/pages/project/ProjectExportPage.groovy

```diff
@@ -18,6 +18,20 @@ class ProjectExportPage extends BasePage {
     By stripJobRefBy = By.name("stripJobRef")
     By stripNameBy = By.xpath("//label[contains(@for,'trip')]")
     By checkBoxBy = By.xpath("//*[@type='checkbox' and not(@id='preserveuuid')]")
+    By allCheckboxBy = By.id('exportAll')
+    By exportJobsCheckboxBy = By.id('exportJobs')
+    By exportExecutionsCheckboxBy = By.id('exportExecutions')
+    By exportConfigsCheckboxBy = By.id('exportConfigs')
+    By exportReadmesCheckboxBy = By.id('exportReadmes')
+    By exportAclsCheckboxBy = By.id('exportAcls')
+    By exportScmCheckboxBy = By.id('exportScm')
+    By tourManagerCheckboxBy = By.id('exportComponents.tours-manager')
```

Comment:
these locators apply only to PRO, it should not be in OSS code

Reply:
Done...